core/src/main/java/nl/nn/adapterframework/scheduler/JobDef.java

```diff
@@ -699,7 +700,11 @@ private void cleanupDatabase(IbisManager ibisManager) {
 				qs.open();
 
 				Message result = qs.sendMessage(Message.nullMessage(), null);
-				log.info("result [" + result + "]");
+				String resultString = result.asString();
+				int numberOfRowsAffected = Integer.valueOf(resultString);
+				if(numberOfRowsAffected > 0) {
+					getMessageKeeper().add("deleted ["+numberOfRowsAffected+"] row(s) from [IBISLOCK] table. It implies that there had been some process(es) finished unexpectedly or failed to complete. Please investigate the log files!", MessageKeeperLevel.WARN);
```

Comment:
It implies that there have been process(es) that finished unexpectedly or failed to complete.